pacman/utilities/file_format_converters/convert_to_java_machine.py

```diff
@@ -0,0 +1,133 @@
+from pacman.utilities import file_format_schemas
+from spinn_utilities.progress_bar import ProgressBar
+from spinn_machine.router import Router
+
+import json
+
+CHIP_HOMOGENEOUS_CORES = 18
```

Comment:
spinnMachine (machine.py).MAX_CORES_IN_MACHINE

no point duplicating.

Reply:
why its there!? not used

Reply:
Agree!